src/slider/Slider.ts

```diff
@@ -174,8 +182,9 @@ export default class Slider extends SliderBase<SliderProperties> {
 				})
 			]),
 			v('output', {
-				classes: this.classes(css.output),
-				for: this._inputId + ''
+				classes: this.classes(css.output, outputIsTooltip ? css.outputTooltip : null),
+				for: this._inputId + '',
```

Comment:
@smhigley: Looks like @tomdye is right, template strings are [fastest now](https://jsperf.com/es6-string-literals-vs-string-concatenation) (previously String concatenation was still fastest; something must've been improved.)